CMakeLists.txt

```diff
@@ -622,10 +681,11 @@ else()
     endif()
   endif()
 
-  set(CMAKE_C_FLAGS_DEBUG "${CMAKE_C_FLAGS_DEBUG} ${DEBUG_FLAGS}")
-  set(CMAKE_CXX_FLAGS_DEBUG "${CMAKE_CXX_FLAGS_DEBUG} ${DEBUG_FLAGS}")
-  set(CMAKE_C_FLAGS_RELEASE "${CMAKE_C_FLAGS_RELEASE} ${RELEASE_FLAGS}")
-  set(CMAKE_CXX_FLAGS_RELEASE "${CMAKE_CXX_FLAGS_RELEASE} ${RELEASE_FLAGS}")
+  set(LOWMEM_FLAGS "--param ggc-min-expand=10 --param ggc-min-heapsize=65536 -O1")
```

Comment:
Clang doesn't support these, so to reduce warning noise in the builds:

```
if (CMAKE_CXX_COMPILER_ID MATCHES "GNU")
  set(LOWMEM_FLAGS "--param ggc-min-expand=10 --param ggc-min-heapsize=65536 -O1")
else()
  set(LOWMEM_FLAGS "-O1")
endif()
```

I would have suggested adding `-Werror=unused-command-line-argument` to the `check_*_compiler_flag`s and running these GC flags through `add_*_flag_if_supported`, but gcc doesn't support that warning :(

Reply:
Those weren't meant to be pushed, will fix.